tools/autograd/derivatives.yaml

```diff
@@ -401,7 +401,7 @@
   self: grad.clone().masked_fill_(self >= other, 0)
   other: grad.clone().masked_fill_(self < other, 0)
 
-- name: mm(Tensor self, Tensor mat2)
+- name: _mm(Tensor self, Tensor mat2)
```

Comment:
I think we might want to specify the derivative on the `mm`. (The derivative definition should be the same even if it uses the addmm dispatch)

Reply:
I didn't want to specify the derivative on `mm` because I'm not sure the backwards functions support sparse